### PR TITLE
Db/addfakedata

### PR DESCRIPTION
**Pull request by arjun0419**
https://trello.com/c/0PzVm7Zv/2-write-fake-data-into-file-in-reasonable-time-l

### CODE REVIEW

File: database/review.js [hunk 1]
```diff
@@ -2,37 +2,42 @@ const mongoose = require('mongoose');
 require('dotenv').config();
 
 // mongoose.connect(process.env[process.env.NODE_ENV]);
+let dbURI = 'mongodb://localhost/Restaurant';
```
Comment (mlbpro417) [inline database/review.js]: should let be const in this case?

File: mockData/seedFunc.js [hunk 2]
```diff
@@ -1,81 +1,83 @@
-const writeReviews = require('./writeAllReviews.js');
-const db = require('../database/review.js');
+Promise = require("bluebird");
 const faker = require('faker');
 const mongoose = require('mongoose');
+const fs = require('fs');
+
+let dbURI = 'mongodb://localhost/Restaurant';
```
Comment (mlbpro417) [inline mockData/seedFunc.js]: I don't think this line is used

File: mockData/seedFunc.js [hunk 3]
```diff
@@ -1,81 +1,83 @@
...
 
-const addtoDB = (review) => {
-  db.insertReview(review, (err) => {
-    if (err) {
-      console.log(err);
-    } else {
-      runner++;
-      if (Number.isInteger(runner / 1000)) {
-        console.log(runner);
+// Write the data to the supplied writable stream one million times.
```
Comment (mlbpro417) [inline mockData/seedFunc.js]: might want to delete comments in this function

File: server/routes/routes.js [hunk 4]
```diff
@@ -1,60 +1,69 @@
...
-      funny: 0,
-      cool: 0,
-    };
-
-    db.insertReview(review, err => (
-      err ? res.sendStatus(500) : res.sendStatus(201)
-    ));
-  }
-});
+    // const review = {
```
Comment (mlbpro417) [inline server/routes/routes.js]: delete comments

File: server/routes/routes.js [hunk 5]
```diff
@@ -1,60 +1,69 @@
...
+    review.useful = 0;
+    review.funny = 0;
+    review.cool = 0;
+    db.insertReview(restaurantId, review, err => (
+      err ? res.sendStatus(500) : res.sendStatus(201)
     ));
   }
 });
 
+// router.put('/restaurants/:restaurantId/reviews/:reviewId', (req, res) => {
```
Comment (mlbpro417) [inline server/routes/routes.js]: delete comments

File: database/review.js (hunk 1 above)
Comment (arjun0419) [reply database/review.js]: changed.

File: mockData/seedFunc.js (hunk 2 above)
Comment (arjun0419) [reply mockData/seedFunc.js]: yes! removed.

File: mockData/seedFunc.js (hunk 3 above)
Comment (arjun0419) [reply mockData/seedFunc.js]: deleted.

File: server/routes/routes.js (hunk 4 above)
Comment (arjun0419) [reply server/routes/routes.js]: deleted

File: server/routes/routes.js (hunk 5 above)
Comment (arjun0419) [reply server/routes/routes.js]: I will leave the put route there, I might enable that route later.

**Comment by arjun0419** [comment]
@mlbpro417 comments cleared. Thanks!

**Pull request merged**